test/utils/dependency-initialiserSpec.js

```diff
@@ -4,7 +4,7 @@ var DependencyInitialiser = require( '../../src/utils/dependency-initialiser' ),
 	LoggerMock = require( '../mocks/logger-mock' ),
 	EventEmitter = require( 'events' ).EventEmitter;
 
-describe( 'dependency-initialiser', function(){
+xdescribe( 'dependency-initialiser', function(){
```

Comment:
this test was swallowing all the errors and the tests in util. Given it doesn't do much I thought it was worth commenting out to make sure everything else works and flesh it out later #technicaldebt